### PR TITLE
docs: fix nextjs doc

### PR DESCRIPTION
**Pull request by santoshyadavdev**
_[Please make sure you have read the submission guidelines before posting an PR](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md#submit-pr)_

> _Please make sure that your commit message follows our format._

> Example: `fix(nx): must begin with lowercase`

### CODE REVIEW

File: docs/shared/next-plugin.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ myorg/
 
 ## See Also
 
-- [Using Next.js](/{{framework}}/plugins_next/overview)
```
Comment (FrozenPandaz) [inline docs/shared/next-plugin.md:63]: This line should stay the same. This should point to a guide that is specific for using Next.js with Nx.

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]
Thank you for your contribution!

I have 1 minor request.

File: docs/shared/next-plugin.md (hunk 1 above)
Comment (santoshyadav198613) [reply docs/shared/next-plugin.md:63]: Then there is a problem, it redirects to the same page, so even if you click nothing happens.
Also, the heading says See Also, which means we are saying something apart from this page, either we should remove this entire See Also section or redirect somewhere else.

Comment (FrozenPandaz) [reply docs/shared/next-plugin.md:63]: I see. Thanks :)

**Review by FrozenPandaz — APPROVED** [review]

**Pull request merged**